readthedocs/api/v2/views/footer_views.py

```diff
@@ -210,6 +211,13 @@ def get(self, request, format=None):
             'version_supported': version.supported,
         }
 
+        # increase the page view count
+        increase_page_view_count.delay(
+            project=context['project'],
```

Comment:
We can't pass the actual model objects, we have to pass the PK or slug. You should probably be running Celery in development to catch these issues: https://docs.readthedocs.io/en/latest/development/standards.html#core-team-standards